backend/alembic/versions/20260722_a9b0c1d2e3f4_move_connector_apps_to_kinds.py

```
@@ -0,0 +1,240 @@
+
+
+def _json_value(value: Any, default: Any) -> Any:
+    return default if value is None else value
+
+
+def _reencrypt_embedded_iv(value: str | None) -> str | None:
+    if not value:
+        return None
+    try:
+        decrypted = decrypt_sensitive_data_with_embedded_iv(value)
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)